Final Project/Client/fiimentor/main/store/Search/createSearchThunk.js

```diff
@@ -0,0 +1,38 @@
+import {Dispatch} from 'redux';
+import Swal from 'sweetalert2';
+import {
+    createSearchAction,
+    createSearchSuccessAction,
+    createSearchErrorAction,
+} from './SearcActions';
+
+import {Context} from '../../Context';
+
+export  const createSearchThunk = (SearchContent)=> async(
```

Comment:
searchPostThunk in loc de createSearchThunk
searchReq in loc de SearchContent, foloseste camelCase la variabile